backend/lib/endpoints/users.js

```diff
@@ -50,6 +50,7 @@ async function routes(app) {
       const userData = {
         ...req.body,
         _id: req.userId,
+        authId: req.user.sub,
```

Comment:
@joaofnds looks like email storage is a yes. For org all looks good - it comes through request body & schema.  For individual user we can do:

```suggestion
        authId,
        email,
```

Both can be destructured from `user` on line 46:

`const { email, email_verified: emailVerified, sub: authId  } = user;` 

After that I have no more suggestions and am happy to get this merged!.

Reply:
If we follow the same behavior as the front-end, maybe it's better to only allow creating user if `email_verified === true` and otherwise return `403 FORBIDDEN` response.
In the frontend, we will check this and redirect to email verification page anyway, so probably better if API follows same behavior, will also be needed for mobile app I guess.

Reply:
@robinv85 doesn't this meet the requirement or am I missing something?

https://github.com/FightPandemics/FightPandemics/blob/9892a582e9f7817ecb0f9a7f28bb38218e3d6ef6/backend/lib/endpoints/users.js#L47

```
      if (!emailVerified) {
        throw app.httpErrors.forbidden("Email address not verified");
      }
```

Reply:
yes that seems good indeed!